application/server/server_test.go

```diff
@@ -1070,18 +1072,50 @@ func runSmokeTest(t *testing.T, repo string, commit string, file1 string, file2
 		if !ok || codeIssueData["hasAIFix"] == false {
 			continue
 		}
-		call, err := loc.Client.Call(ctx, "workspace/executeCommand", sglsp.ExecuteCommandParams{
+		call, e := loc.Client.Call(ctx, "workspace/executeCommand", sglsp.ExecuteCommandParams{
```

Comment:
question: why the rename?

Reply:
The lint [complained](https://github.com/snyk/snyk-ls/actions/runs/8251531711/job/22568804913) about: `shadow: declaration of "err" shadows declaration at line 996 (govet)` and `cyclomatic complexity for runSmokeTest`. So first I addressed the shadow declaration but later I extracted the code related to the Feature Flag to `checkFeatureFlagStatus` so I guess I can undo the renaming or `err`